tests/integration/Dockerfile.integration

```diff
@@ -1,6 +1,6 @@
 ARG VPP_IMG=ligato/vpp-base
 
-FROM golang:1.15 as build
+FROM golang:1.17 as build
```

Comment:
Why is this version bump needed? 
Also this is hardcoded and could go easily out of sync with golang version used elsewhere. I know it was hardcoded also before this PR, however it could be done better.

Reply:
In #1833 all occurrences of golang version were updated to 1.17, except this one. So I updated it also here for consistency.

Reply:
So this is a little bit unrelated to VPP 22.02 and it is just version catch-up fix. Normally, i would expect another PR for unrelated changes. However, this change is quite little and IMHO it can be passed with these bigger changes about VPP 22.02.